test/unit/builtins/numbers.spec.ts

```diff
@@ -69,6 +69,13 @@ test.each([
     util.testExpressionTemplate`(${value}).toString(2)`.expectToEqual(luaNativeSpecialNumString);
 });
 
+const toFixedRadixes = [undefined, 0, 1, 2, 5, 10, 80];
+const toFixedValues = [-1, 0, 1, /* 1.5 fails, */ 1024, 1.2, 1e19, 1e22];
```

Comment:
Why does 1.5 fail?

Reply:
Because JS is rounding it up to `2` but the used Lua implementation is rounding it down to `1`

Possibly https://stackoverflow.com/questions/64562060/why-does-formatting-a-number-round-differently-in-luajit-vs-lua

Reply:
> ```
> // Differences in math implementations between JS & Lua cause inaccuracies
> // therefore test input numbers are "carefully" selected to always match accuratly.